cockpit/src/subscriptions-register.jsx

```diff
@@ -101,7 +104,53 @@ class SubscriptionRegisterDialog extends React.Component {
 
 
         let credentials;
+        let organizations;
         if (this.props.register_method === "account") {
+            // Show the list of available organizations, when user have to choose one org from the list
+            if (subscriptionsClient.orgList.length > 0) {
+                let orgs_list = [];
+                let org;
+                // Javascript/JSX is doomed, when I have to do it in this way :-/
+                let disabled=true;
+                let is_placeholder=true
+                // It seems that FormSelect does not have regular placeholder :-(
+                orgs_list.push(
+                    <FormSelectOption
+                        value=''
+                        label={_('Choose an organization')}
+                        isDisabled={disabled}
```

Comment:
It's much more nice and a common pattern to not specify the values of truthy properties.
So that would get:

``` js
isDisabled
```
rather than 
``` js
isDisabled={disabled}
```
since disabled is constant and always true.